npm/private/npm_translate_lock.bzl

```diff
@@ -92,8 +83,6 @@ write_source_files(
 ```
 
 Then in `WORKSPACE`, load from that checked-in copy or instruct your users to do so.
-In this case, the aliases are not created, so you get only the `npm_import` behavior
```

Comment:
I'm not sure if we still need something here to warn "hey the @npm_deps repo won't exist in this case"

Reply:
I think we still need that since we have the bin aliases.

Reply:
https://github.com/aspect-build/rules_js/blob/506d26d59b356514d0a0a38ea33fc7d6eacc6677/npm/private/npm_translate_lock.bzl#L808-L839 we still have bin aliases.